OWNERS_ALIASES

```diff
@@ -159,12 +152,24 @@ aliases:
     - aasmall
     - joelsmith
     - cji
+    - jaybeale
   ug-big-data-leads:
     - foxish
     - erikerlandson
     - liyinan926
-## BEGIN CUSTOM CONTENT
-  steering-committee:
+  committee-code-of-conduct:
+    - jdumars
+    - parispittman
+    - carolynvs
+    - eparis
+    - bradamant3
+  committee-product-security:
+    - philips
+    - cjcullen
+    - tallclair
+    - liggitt
+    - joelsmith
+  committee-steering:
```

Comment:
The committee aliases aren't really used anywhere in this repo so moving `committee` as a suffix is a no-op.